python-sdk/nuscenes/eval/panoptic/panoptic_track_evaluator.py

```diff
@@ -259,11 +259,54 @@ def get_ptq(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
 
         ptq_all = ((iou - ids) / tp_eps) * (tp / tp_half_fp_half_fn_eps)  # Calculate PTQ of all classes.
         soft_ptq_all = ((iou - soft_ids) / tp_eps) * (tp / tp_half_fp_half_fn_eps)  # Calculate soft-PTQ of all classes.
-        mean_ptq = ptq_all[self.include].mean()  # Mean PTQ over all classes except ignored classes.
-        mean_soft_ptq = soft_ptq_all[self.include].mean()  # Mean soft-PTQ over all classes except ignored classes.
+
+        # Get classes that at least has 1 ground truth instance, and is included in self.include.
+        ground_truths = tp_eps + fn
+        valid_classes = ground_truths > 0.5
```

Comment:
Is `ground_truths > 0.5` the same as `ground_truths > self.iou_thr`?

Reply:
@whyekit-motional That's not the same, 0.5 is a random threshold to make sure there is at least 1 ground truth objects. maybe I can change to 1.0 to reduce the confusion a bit.

Reply:
I think leaving it as `0.5` is fine; I was just curious about the calculation, haha

I added a `<nit>` but I think github removed it :man_shrugging:

Reply:
Let's add a comment to explain that

Reply:
Does the tp_eps make a difference?

Reply:
updated the comments for this threshold explanation. tp_eps is 1e-15, it is far less than the 0.5 threshold. so it has no impact on this.

Reply:
Then let's use tp

Reply:
ok